librz/bin/bobj.c

```diff
@@ -685,10 +685,189 @@ RZ_API ut64 rz_bin_object_get_vaddr(RzBinObject *o, ut64 paddr, ut64 vaddr) {
 	return paddr;
 }
 
-RZ_API RzBinAddr *rz_bin_object_get_special_symbol(RzBinObject *o, RzBinSpecialSymbol sym) {
+RZ_API const RzBinAddr *rz_bin_object_get_special_symbol(RzBinObject *o, RzBinSpecialSymbol sym) {
 	rz_return_val_if_fail(o, NULL);
 	if (sym < 0 || sym >= RZ_BIN_SPECIAL_SYMBOL_LAST) {
 		return NULL;
 	}
 	return o ? o->binsym[sym] : NULL;
 }
+
+/**
+ * \brief Get list of \p RzBinAddr representing the entry points of the binary object.
+ */
+RZ_API const RzList *rz_bin_object_get_entries(RzBinObject *obj) {
```

Comment:
Ownership mark, since all of them are borrow

Reply:
As said above, I think the `const RzList *` already makes it clear that the list is not owned by the caller, no?

Reply:
Well, technically you are right, but then we have 2 different ways of doing the same. It isn't good probably. I wonder if we should just define `RZ_BORROW` as `const`.

Reply:
I think RZ_BORROW is especially useful when you don't have something `const` and you can't make it const for whatever reason, but you still want the caller to not own the object.